cpp/tests/experimental/renumbering_test.cpp

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governin_from_mtxg permissions and
+ * limitations under the License.
+ */
+
+#include <utilities/high_res_clock.h>
+#include <utilities/base_fixture.hpp>
+#include <utilities/test_graphs.hpp>
+#include <utilities/test_utilities.hpp>
+#include <utilities/thrust_wrapper.hpp>
+
+#include <cugraph/algorithms.hpp>
+#include <cugraph/experimental/graph.hpp>
+#include <cugraph/experimental/graph_functions.hpp>
+#include <cugraph/experimental/graph_view.hpp>
+
+#include <raft/cudart_utils.h>
+#include <raft/handle.hpp>
+#include <rmm/device_uvector.hpp>
+#include <rmm/mr/device/cuda_memory_resource.hpp>
+
+#include <gtest/gtest.h>
+
+#include <algorithm>
+#include <iterator>
+#include <limits>
+#include <numeric>
+#include <random>
+#include <vector>
+
+// do the perf measurements
+// enabled by command line parameter s'--perf'
+//
+static int PERF = 0;
+
+struct Renumbering_Usecase {
+  bool check_correctness{true};
+};
+
+template <typename input_usecase_t>
+class Tests_Renumbering
+  : public ::testing::TestWithParam<std::tuple<Renumbering_Usecase, input_usecase_t>> {
+ public:
+  Tests_Renumbering() {}
+  static void SetupTestCase() {}
+  static void TearDownTestCase() {}
+
+  virtual void SetUp() {}
+  virtual void TearDown() {}
+
+  template <typename vertex_t, typename edge_t, typename weight_t>
+  void run_current_test(Renumbering_Usecase const& renumbering_usecase,
+                        input_usecase_t const& input_usecase)
+  {
+    raft::handle_t handle{};
+    HighResClock hr_clock{};
+
+    std::vector<vertex_t> h_original_src_v{};
+    std::vector<vertex_t> h_original_dst_v{};
+    std::vector<vertex_t> h_final_src_v{};
+    std::vector<vertex_t> h_final_dst_v{};
+
+    rmm::device_uvector<vertex_t> src_v(0, handle.get_stream_view());
+    rmm::device_uvector<vertex_t> dst_v(0, handle.get_stream_view());
+    rmm::device_uvector<vertex_t> renumber_map_labels_v(0, handle.get_stream_view());
+    vertex_t number_of_vertices{};
+    edge_t number_of_edges{};
+    bool symmetric{};
```

Comment:
This variable is not used after the assignment in line 81. So, you can remove this variable and replace

```std::tie(src_v, dst_v, std::ignore, number_of_vertices, symmetric) =```
with
```std::tie(src_v, dst_v, std::ignore, number_of_vertices, std::ignore) =```

Reply:
Done